src/Adapter/Product/Image/Repository/ProductImageMultiShopRepository.php

```diff
@@ -211,14 +211,23 @@ public function create(ProductId $productId, ShopConstraint $shopConstraint): Im
     }
 
     /**
-     * @param Image $image
-     * @param ShopId[] $shopsToRemoveImageFrom
+     * @param ImageId $imageId
+     * @param ShopId[] $shopIds
      *
      * @return void
      */
-    public function deleteFromShops(Image $image, array $shopsToRemoveImageFrom)
+    public function deleteFromShops(ImageId $imageId, array $shopIds): void
```

Comment:
these changes wasn't required, I just adjusted it a little so it is more usable and similar to whats done in product repository

Reply:
Isn't it a BC Break ?

Reply:
Nope it's all part of experimental code, no backward policy until the page is finished